src/components/RenderRouter/BlogReader.scss

```diff
@@ -162,6 +180,24 @@
         box-shadow: none !important;
     }
     
+    .download-custom {
```

Comment:
Move any common styles to outside of the media queries, that way you don't need to duplicate them. Ideally the styles in the media queries are only the parts affected by the screen size.